CHANGES.md

```diff
@@ -1,3 +1,128 @@
+## Version 0.7.0
+
+> 2016 August 2
+
+Version 0.7.0 is the seventh Cats release, and includes several major rearrangements and changes to names.
+
+### Migration notes
```

Comment:
Thank you for adding this section @travisbrown. I think this will be really helpful for people.